generators/python/tests/utils/test_construct_type.py

```diff
@@ -73,7 +76,8 @@ def test_construct_unset() -> None:
         "map": {"1": "string"},
         "union": {"type": "square", "id": "string", "length": 1.1},
         "undiscriminated_union": {"id": "string2", "length": 6.7},
-        "enum": "red"
+        "enum": "red",
+        "any": "something here"
```

Comment:
is this to check additional props?

Reply:
nvm i see you added to the fern def thanks

Reply:
Yeah! Just adding an any type